### ISSUE TITLE
Unable to add a new keypair using cli

### ISSUE THREAD

**Issue by rapsealk** [lablup/backend.ai#1657]
### What Operating System(s) are you seeing this problem on?

Linux (x86-64)

### Backend.AI version

60e4bde7765dce35697a6a9dd21accc11794abd9

### Describe the bug

Keypair add command fails with the error message below:
```sh
$ ./backend.ai --output=json admin keypair add test0@lablup.com default
{
  "ok": false,
  "msg": "unexpected error: Could not locate column in row for column 'groups_name'",
  "item_name": "keypair",
  "action_name": "add"
}
```

https://github.com/lablup/backend.ai/blob/60e4bde7765dce35697a6a9dd21accc11794abd9/src/ai/backend/manager/models/keypair.py#L217

### To Reproduce

1. Create a new user
```sh
$ ./backend.ai admin user add default username@backend.ai 0000
```
2. Create a new keypair for the user
```sh
$ ./backend.ai admin keypair add username@backend.ai default
```

### Expected Behavior

Successfully create a new keypair for the given user.

### Anything else?

It seems to be related to GraphQL or database fields.

**Comment by achimnol** [comment]
This is being handled in #1360, and this particular fix is already cherry-picked to #1632.
So, #1661 is a duplicate of #1360 and #1632.
@adrysn Please review and apply #1632 ASAP.

**Comment by achimnol** [comment]
I'm going to merge this as a separate patch, as it's a simple single-liner.
